code/__HELPERS/unsorted.dm

```diff
@@ -198,6 +198,8 @@ Turf and target are separate in case you want to teleport some distance from a t
 
 	while(loop && safety < 5)
 		if(C && C.prefs.custom_names[role] && !safety && !banned)
+			if(!C.prefs)
```

Comment:
Remove these changes, the entire proc is reworked in tgui prefs

Reply:
But my merge conflicts